packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -1,18 +1,34 @@
 /*
- * Copyright (C) 2017 TypeFox and others.
+ * Copyright (C) 2017-2018 TypeFox and others.
  *
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  */
 
-import { injectable } from "inversify";
-import { FILE_NAVIGATOR_ID, FileNavigatorWidget } from './navigator-widget';
+import { injectable, inject, postConstruct } from "inversify";
 import { AbstractViewContribution } from '@theia/core/lib/browser/shell/view-contribution';
+import { CommandRegistry, MenuModelRegistry } from "@theia/core/lib/common";
+import { Navigatable, ITreeNode, ISelectableTreeNode, Widget, KeybindingRegistry } from "@theia/core/lib/browser";
+import { SHELL_TABBAR_CONTEXT_MENU } from "@theia/core/lib/browser";
+import { FILE_NAVIGATOR_ID, FileNavigatorWidget } from './navigator-widget';
+import { FileNavigatorModel } from "./navigator-model";
+import { FileNavigatorPreferences } from "./navigator-preferences";
+
+export namespace FileNavigatorCommands {
+    export const REVEAL_IN_NAVIGATOR = {
+        id: 'navigator.reveal',
+        label: 'Reveal in Files'
+    };
+}
 
 @injectable()
 export class FileNavigatorContribution extends AbstractViewContribution<FileNavigatorWidget> {
 
-    constructor() {
+    protected navigatorModel: FileNavigatorModel;
```

Comment:
it is already accessible via `this.widget`, no need to add it to `FileNavigatorContribution` state

Reply:
ok